RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedProducer.cc

```diff
@@ -283,78 +256,93 @@ void ElectronSeedProducer::fillDescriptions(edm::ConfigurationDescriptions& desc
     psd2.add<double>("deltaEtaRegion", 0.1);
     psd2.add<double>("ptMin", 1.5);
     psd2.add<double>("originRadius", 0.2);
-    psd2.add<edm::InputTag>("VertexProducer", edm::InputTag("dummyVertices"));
+    psd2.add<edm::InputTag>("VertexProducer", {"dummyVertices"});
     psd0.add<edm::ParameterSetDescription>("RegionPSet", psd2);
 
-    psd0.add<double>("PhiMax2B", 0.002);
+    // steering
+    psd0.add<edm::InputTag>("initialSeeds", {""});  //keep for be compatibility
+    psd0.add<std::vector<edm::InputTag>>("initialSeedsVector", {});
+    psd0.add<bool>("preFilteredSeeds", false);
+    psd0.add<bool>("useRecoVertex", false);
+    psd0.add<edm::InputTag>("vertices", {"offlinePrimaryVerticesWithBS"});
+    psd0.add<edm::InputTag>("beamSpot", {"offlineBeamSpot"});
+    psd0.add<bool>("dynamicPhiRoad", true);
+
+    // specify where to get the hits from
+    psd0.add<edm::InputTag>("measurementTrackerEvent", {"MeasurementTrackerEvent"});
```

Comment:
is this parameter still used?

Reply:
Yes, this one is still used. Not in the current configuration thought, but at least the plugin could theoretically be configured such that it is actually used in a meaningful way.